tests/vppoutputencode.cpp

```diff
@@ -25,6 +25,28 @@
 #include "common/common_def.h"
 #include "vppoutputencode.h"
 
+EncodeParams::EncodeParams()
+    : rcMode(RATE_CONTROL_CQP)
+    , initQp(26)
+    , bitRate(0)
+    , fps(30)
+    , ipPeriod(1)
+    , ipbMode(1)
+    , kIPeriod(30)
+    , fourcc(0)
```

Comment:
All parameters here are default values, but fourcc equal to zero is meanlingless. Would better set it to be nv12 or I420(if I420 supported)